src/main/java/edu/cornell/kfs/module/purap/document/CuPaymentRequestDocument.java

```diff
@@ -208,28 +226,6 @@ public void customizeExplicitGeneralLedgerPendingEntry(
         }
     }
     
-    public void doActionTaken(final ActionTakenEvent event) {

```

Comment:
Just to confirm, was this method's removal needed for the first part of the fix (the part that Daniela had prepared a fix for)? Or was it needed for the double-extract part? Or should it be left in?

Reply:
Please compare this method to the method of the same name in overlay file PaymentRequestDocument. I believe that it is exactly the same code. The difference IMHO are just in the way that the service calls are being defined/made and the manner in which the constants are being stored/retrieved.

Reply:
Thanks for the clarification. I agree that the logic and effects are the same, so I'm fine with this method being removed.